packages/stryker-html-reporter/src/templates/sourceFile.tsx

```diff
@@ -7,7 +7,15 @@ import { ScoreResult, SourceFile, MutantResult, MutantStatus } from 'stryker-api
 import Breadcrumb from '../Breadcrumb';
 import { mutantTable } from './mutantTable';
 
+export interface NumberedMutant {
+    index: number;
+    mutant: MutantResult;
+}
+
 export function sourceFile(result: ScoreResult, sourceFile: SourceFile | undefined, mutants: MutantResult[], breadcrumb: Breadcrumb, thresholds: MutationScoreThresholds) {
+    const numberedMutants = _
+        .sortBy(mutants, m => m.range[0] * 10000 + m.range[1] * -1)
```

Comment:
What? Why not just order by the start range?

Reply:
Lol, i think it is a left over from when we did ordering based on position (line + column). I'll replace it.